src/main/java/org/example/api/SystemApi.java

```diff
@@ -0,0 +1,36 @@
+package org.example.api;
+import org.example.model.Directory;
+import org.example.model.MyFile;
+
+public class SystemApi {
+    private final IFileApi api = new FileService();
+
+    public MyFile addFile(Directory directory, MyFile myFile) {
+        return api.addFile(directory, myFile);
+    }
+
+    public MyFile removeFile(Directory directory, String fileName) {
+
+        return api.removeFile(directory, fileName);
+    }
+
+    public MyFile removeFile(String fileName) {
+        return api.removeFile(fileName);
+    }
+
+    public MyFile getFile(String fileName) {
```

Comment:
I think we need to add id/fileId field to the File. what if I have 2 files with same names in differet directories? This method seems like it will return any of them which it finds first. Filename can't be a unique identifier across all directories as 2 files with same names can exist in different directories. We can add id field which will be unique for each created file, you can have a look at UUID class and generate unique ids with it.

Reply:
I didn't implement UUID as it was difficult for later usage for accessing from tests, I did the ID part with long values.